src/components/AddAnswer.jsx

```diff
@@ -48,9 +49,11 @@ function AddAnswer({ question }) {
           <input type="email" name="email" placeholder="Example: jack@email.com" className="form-input" maxLength={60} required />
           <p className="text-sm text-neutral-400">For authentication reasons, you will not be emailed</p>
         </label>
```

Comment:
It looks like this might be missing an opening `<label> `tag

Reply:
It's on the line above, just not shown in the GitHub preview.